examples/documentation.html

```diff
@@ -899,6 +899,26 @@ <h3 class="option"><span class="expand all">[ <a href="javascript:$('#optionlist
             desc: "For sunburst only: specifies wether the leaf elements of a parent get the same color as the parent or if they are colored individually.",
             default: "true",
             examples: [true, false]
+        },
+        showLabels: {
+            desc: "For sunburst only: specifies if labels are displayed. The labels are centered on the arcs. The actual output can be adjusted by specifying the 'labelText' function.",
+            default: "false",
+            examples: [true, false]
+        },
+        labelText: {

```

Comment:
Just checked the pie ... labelFormat is marked as deprecated:

    // deprecated after 1.7.1
        labelFormat: {get: function(){ return valueFormat;}, set: function(_) {
            valueFormat=_;
            nv.deprecated('labelFormat','use valueFormat instead');
        }},)

And just maps to valueFormat. For my purpose I need some more fine-tuning for the label than just specifying the way the value should be displayed. Should I remove the whole option than or do you suggest another name to make the difference obvious.